src/effect.cpp

```diff
@@ -1547,9 +1547,9 @@ void NewEffect(int A, const Location_t &Location, float Direction, int NewNpc, b
         ne.Location.SpeedY = -2 - dRand() * 10;
         ne.Location.SpeedX = dRand() * 8 - 4;
         ne.Frame = 0;
-        if((iRand() % 2) == 1)
+        if(iRand(2) == 0)
```

Comment:
Why 0 and not 1? But seems, doesn't matter, just the 1/2 chance this will be true.

Reply:
Just silly micro-optimization; I think that 1 translates to a subtract and a test, and 0 translates to just a test. Does that sound right to you?

Reply:
ye